Vagrantfile

```diff
@@ -5,7 +5,7 @@ Vagrant.configure('2') do |c|
   server_post_up = <<-EOS.gsub(/^[[:blank:]]{4}/, '')
     Now run the following commands to get going:
 
-        simp config -a /vagrant/answers.yaml
+        simp config -a /vagrant/simp_config_answers.yaml
```

Comment:
This was originally just the simp config answers file, taken from /root/.simp/simp_conf.yaml